File: app/src/main/java/fr/insapp/insapp/models/NotificationUser.java
package fr.insapp.insapp.models;

import com.google.gson.annotations.SerializedName;

/**
 * Created by thomas on 18/07/2017.
 */

public class NotificationUser {

    @SerializedName("ID")
    private String id;

    @SerializedName("userid")
    private String userId;

    @SerializedName("token")
    private String token;

    @SerializedName("os")
    private String os;

    public NotificationUser(String id, String userId, String token, String os) {
        this.id = id;
        this.userId = userId;
        this.token = token;
        this.os = os;
    }

    public String getId() {
        return id;
    }

    public String getUserId() {
        return userId;
    }

    public String getToken() {
        return token;
    }

    public String getOs() {
        return os;
    }
}

File: app/src/main/java/fr/insapp/insapp/models/NotificationUser.kt
package fr.insapp.insapp.models

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

/**
 * Created by thomas on 18/07/2017.
 * Kotlin rewrite on 30/08/2019.
 */

@Parcelize
data class NotificationUser(
    @SerializedName("ID") val id: String?,
    @SerializedName("userid") val userId: String,
    @SerializedName("token") val token: String,
    @SerializedName("os") val os: String
): Parcelable
